integration-libs/opf/base/root/model/opf.model.ts

```diff
@@ -51,7 +58,7 @@ export interface PaymentBrowserInfo {
   screenHeight?: number;
   screenWidth?: number;
   userAgent?: string;
-  timeZoneOffset?: number;
+  timezoneOffset?: number;
```

Comment:
please ignore, it is to be compatible with test server, will be reverted once Bamboo will fix it.
ref: [CXSPA-4152](https://jira.tools.sap/browse/CXSPA-4152)
OPF Hosted-fields: revert timezoneOffset property typo